src/contracts/core/RewardsCoordinator.sol

```diff
@@ -274,6 +278,47 @@ contract RewardsCoordinator is
         emit ClaimerForSet(earner, prevClaimer, claimer);
     }
 
+    /**
+     * @notice Sets the commission for a specific operator for a specific operatorSet (avs, operatorSetId)
+     * @param operator The operator address
+     * @param avs The address of the AVS
+     * @param operatorSetId The operatorSetId of the AVS
+     * @param commissionBips The commission in bips for the operator, must be <= MAX_COMMISSION_BIPS
+     * @return effectTimestamp The timestamp at which the operator commission update will take effect
+     * @dev Only callable by the operator
+     */
+    function setOperatorCommissionBips(
+        address operator,
+        address avs,
+        uint32 operatorSetId,
+        uint16 commissionBips
+    ) external returns (uint32) {
+        require(
+            msg.sender == operator,
+            "RewardsCoordinator.setOperatorCommissionBips: caller is not operator"
+        );
+        require(
+            commissionBips <= MAX_COMMISSION_BIPS,
+            "RewardsCoordinator.setOperatorCommissionBips: commissionBips too high"
+        );
+        uint32 effectTimestamp = uint32(block.timestamp + OPERATOR_COMMISSION_ACTIVATION_DELAY);
```

Comment:
This can be unchecked, overflow is not reasonably possible.

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/627/commits/68478ee59925e8e2cbb5fb1bfc348079fe6cb614